src/classes/ADDR_Addresses_TDTM.cls

```diff
@@ -290,7 +290,7 @@ public class ADDR_Addresses_TDTM extends TDTM_Runnable {
         set<Id> setHHId =  mapAccIdAddr.keySet();
         list<Contact> listCon = [select Id, is_Address_Override__c, Current_Address__c, AccountId 
             from Contact where 
-                (Account.SYSTEM_AccountType__c = :CAO_Constants.HH_ACCOUNT_PROCESSOR or Account.SYSTEM_AccountType__c = :CAO_Constants.ADM_ACCOUNT_PROCESSOR)
+                (Account.RecordType.Name = :CAO_Constants.HH_ACCOUNT_PROCESSOR or Account.RecordType.Name = :CAO_Constants.ADM_ACCOUNT_PROCESSOR)
```

Comment:
You know, if we were smart, we'd be storing this as the id instead of the name... am I crazy for thinking that's safer?